Yoast/wordpress-seo
# Add tooling to internationalize the configuration wizard

## Summary

This adds the tooling to internationalize the configuration wizard. Combine the POT files of Yoast SEO, YoastSEO.js, and yoast-components to have one project for translators to translate. Then create JSON files specifically for the configuration wizard.

## Relevant technical choices:

* Use i18n-calypso to parse because `xgettext` cannot parse ES6 classes. A combination with babel is possible, but I didn't want to reinvent the wheel.
* Use `file_get_contents` to get the contents of the JSON files to give to the JS because we did this with YoastSEO.js and haven't had reports of the translation not working because of server configurations.
* Don't commit the actual JSON files because the strings will change and they can be committed when releasing.

## Test instructions

This PR can be tested by following these steps:

* Make sure `grunt build:i18n` works.
* Make sure `grunt build:i18n:makepot` works.
* After doing `grunt build:i18n` and setting WordPress to a translated language such as dutch or danish the next button is translated in the configuration wizard.

Fixes https://github.com/Yoast/yoast-components/issues/35
Requires https://github.com/Yoast/yoast-components/pull/64.
